src/screens/chats/index.js

```diff
@@ -12,26 +15,44 @@ class Chats extends PureComponent {
     }
 
     handleOpenChat = (chat) => {
-        const { onChatOpen } = this.props
-        onChatOpen(chat)
+        const { onChatOpen } = this.props;
+        onChatOpen(chat);
+    }
+
+    handleNavigateToUsers = () => {
+        const { onNavigate } = this.props;
+        onNavigate(ROUTES.USERS)
+    }
+
+    handleLogout = () => {
```

Comment:
Please, write in one line, use destructuring here is redundantly. For all functions like this.